strabo/static/js/map.js

```diff
@@ -37,7 +37,24 @@ $(document).ready(function(){
     //set_map_click(map);
 
     set_geolocation(map);
+
+});
+
+$(document).keyup(function(e) {
+    if (e.keyCode==27) { //if ESC key is hit
```

Comment:
I would put in a variable assignment here to better convey the meaning of what is happening. Comments are good, but this will  help as well.

    var KEYCODE_ESC = 27;